README.md

```diff
@@ -93,3 +93,7 @@ muscle: 1
 action: 'move to'
 setpoint: 10.0" 
 ```
+### Units
+- Position controller ('move to' command) expects the setpoint in radians 
+- actuarorPosition and tendonDisplacement are displayed in encoder ticks
+- 1 rad ~ 18000 ticks
```

Comment:
Ahh, found what it was deep in the back of my mind:

- Gear box is correct at 53:1
- Encoder tics is also correct at 512 tics per rotation, BUT being a quadrature encoder a tic is actually 4 tics/pulses:

512*53*4=108544 tics per rotation 

108544/(2+3.1415) = 17275.8236 or ~18000